test/e2e/mcd_test.go

```diff
@@ -70,19 +70,13 @@ func createMCFile(path, content string, mode int) ignv2_2types.File {
 	}
 }
 
-func TestMCDeployed(t *testing.T) {
-	cb, err := common.NewClientBuilder("")
-	if err != nil {
-		t.Errorf("%#v", err)
-	}
-	mcClient := cb.MachineConfigClientOrDie("mc-file-add")
-	k := cb.KubeClientOrDie("mc-file-add")
-
+func createMCToAddFile(filename, data string) *mcv1.MachineConfig {
 	// create a dummy MC
-	mcName := fmt.Sprintf("00-0add-a-file-%s", uuid.NewUUID())
+	mcName := fmt.Sprintf("%s", uuid.NewUUID())
```

Comment:
That uuid may sort before the built in MCs right?  How about `mctest-addfile-$(date +'%s')` (in shell terms)?

Reply:
fixed thanks!